### ISSUE TITLE
Missing characters in my name on the website

### ISSUE THREAD

**Issue by Ra5tko** [keybase/keybase-issues#2440]
So my name is Rastko Đorđević (notice UTF-8 characters Đ, đ and ć), but it is displayed as Rastko orevi. My keybase username is ra5tko.

**Comment by malgorithms** [comment]
hmm, this should work! What did you use to edit your full name? The website or an app? If website, what browser/version/os? I just changed my middle name in Chrome 51/OSX and it worked:

![image](https://cloud.githubusercontent.com/assets/614943/17488687/8b4eef60-5d68-11e6-87be-32f1b66df127.png)

clearly yours got stripped somehow, though. I wonder how...it's not a display problem, since it's missing in the DB, so I'm mostly curious how you posted it.

thanks!

**Comment by Ra5tko** [comment]
It seems to be the issue when creating my account. I just tried changing my name in Chrome 52/Windows 10 and I have no problem changing the name, but it rather seems to be that the characters striped off when creating my account. My guess would be that he letters got striped off when submitting the data to SQL during the account creation. :/
Side note:
It seems that the font you are using - Lato doesn't support Đ, đ, Ć etc. so it uses Helvetica Neue, not a big problem, but is just seems weird compared to other letters. :P

Thanks! :D

**Comment by malgorithms** [comment]
so weird...we don't ask you for your full name when joining, we just land you on your profile page, where you can change it. Which means it's the same form when you're editing it later.

I just made a new account and edited the name right away to yours and I hit no problem. 

Very confusing! If you can think of anything that was unusual about your signup process, please update this ticket.

**Comment by Ra5tko** [comment]
You are right, just sent an invitation to a family member and she had no problem when editing her name after account creation. Really strange. I'll let you know if I found something.
